### PR TITLE
Dev UI qute config - switch to strict rendering

### LINKED ISSUE #18485: Dev UI qute config - switch to strict rendering
This is a follow-up of https://github.com/quarkusio/quarkus/pull/18416. Strict rendering breaks the main page. We should fix the template, verify all other pages and add smoke test for every extension.

### PR DESCRIPTION
**Pull request by mkouba**
- resolves #18485
- also added smoke tests for most of the extensions that provide
qute-based UI

### CODE REVIEW

**Comment by mkouba** [comment]
FTR I tried to add a smoke test for most of the extensions that contribute a qute template in the `integration-tests/devmode`. Note that those tests cannot be placed under the relevant `-deployment` module because then the DEV UI processor would not be able to locate the template resources correctly.

OK - test added
DS - not added due to difficult setup
NN - test not needed

| Extension | Status |
| --- | --- |
| flyway | DS |
| grpc | OK |
| container-image | OK |
| infinispan-client | DS |
| kubernetes-openshift | DS |
| qute | OK |
| scheduler | OK |
| liquibase | DS |
| smallrye-health | NN |
| arc | OK |
| smallrye-reactive-messaging| OK |
| vertx-http | OK |
| cache | OK |
| smallrye-openapi | NN |
| smallrye-graphql | NN |
| hibernate-search-orm-elasticsearch | DS |
| resteasy-reactive | OK |

File: extensions/vertx-http/deployment/src/main/resources/dev-templates/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
        {#actionableExtension it/}
    {/each}
    </div>
-   {#if nonActionableExtensions.size() > 0}
+   {#if nonActionableExtensions}
```
Comment (mkouba) [inline extensions/vertx-http/deployment/src/main/resources/dev-templates/index.html:27]: An empty collection is considered `falsy`...

File: extensions/vertx-http/deployment/src/main/resources/dev-templates/io.quarkus.quarkus-vertx-http/config.html [hunk 2]
```diff
@@ -146,9 +146,7 @@ <h5>
                         {/if}
                     </td>
                     <td>
-                        {#if item.description.fmtJavadoc && item.description.fmtJavadoc != 'NOT_FOUND'}
-                            {item.description.fmtJavadoc}
-                        {/if}    
+                       {item.description.fmtJavadoc??}
```
Comment (mkouba) [inline extensions/vertx-http/deployment/src/main/resources/dev-templates/io.quarkus.quarkus-vertx-http/config.html:149]: This is an equivalent of `{item.description.fmtJavadoc.or(null)}` and `null` values produce no output.

File: extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/actionableExtension.html [hunk 3]
```diff
@@ -3,24 +3,24 @@
         <div class="card-header bg-primary text-light guide-padding-right">
             <div class="row">
                 <div class="col-md-10">
-                    {it['name']}
+                    {it.name}
```
Comment (mkouba) [inline extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/actionableExtension.html:6]: The bracket notation is not needed here and is not very readable...

**Review by gastaldi — APPROVED** [review]

File: extensions/scheduler/deployment/src/main/resources/dev-templates/tags/scheduleInfo.html [hunk 4]
```diff
@@ -5,6 +5,6 @@
 {/if}
 {#if it.delay > 0}
   (with delay {it.delay} {it.delayUnit.toString.toLowerCase})
-{#else if !schedule.delayed.empty}
+{#else if !it.delayed.empty}
```
Comment (gsmet) [inline extensions/scheduler/deployment/src/main/resources/dev-templates/tags/scheduleInfo.html:8]: Yeah so that was useful :).

**Review by gsmet — APPROVED** [review]

**Pull request merged**